executor/common_windows.h

```diff
@@ -111,3 +107,26 @@ static int do_sandbox_none(void)
 	return 0;
 }
 #endif
+
+static void use_temporary_dir(void)
+{
+	char tmpdir_template[] = "./syzkaller.XXXXXX";
+	char* tmpdir = mktemp(tmpdir_template);
+
+	CreateDirectory(tmpdir, NULL);
```

Comment:
I am not sure about the `chdir` part in the other use_temporary_dir implementation, but at the very least we need the `chdir` here. This function needs to chdir, it does not even return the temp dir name, so there is not way caller can do anything else with the new temp dir.

Reply:
I've made the change. I did not notice this.